website/app/styles/doc-components/table-of-contents.scss

```
@@ -57,8 +57,7 @@ $doc-toc-item-inset: 8px;
     display: none;
   }
 
-  // it's using a `<detail>` HMTL tag
-  [open] > & {
+  &.open {
```

I know it's scoped, but could we follow BEM and use a modifier instead? then we can even unnest it
```suggestion
  &.doc-table-of-contents__button--open {
```